kuzzleio/kuzzle-admin-console
# KZL 305 - Visualize documents on a map

## What does this PR do ?

This PR add a ViewMap component to display documents on a map.  
The component use the documents `geo_point` to display markers on the map.  
The user can choose which `geo_point` to use with a dropdown.  
When you click on a marker, you can see the document content

### How should this be manually tested?

  - Step 1 : Execute [this script](https://gist.github.com/Aschen/c7e7509aaa598dbe7d5b7efd4f3d4f68) to have some geolocated data
  - Step 2 : Go to the collection view and select map view in the top right corner
  - Step 3 : Select some marker, you should see information about the document
  - Step 4 : Change the selected geopoint to `emplacement` instead of `location`

### Other changes



### Boyscout

  - fix bug in the throttle that cause inconcistency between the quick filter content and the url

### Screenshots (if appropriate)


![image](https://user-images.githubusercontent.com/4447392/44723377-14948f80-aad0-11e8-8884-ac4ff5847d76.png)
